src/pages/inventory/inventory.scss

```diff
@@ -0,0 +1,3 @@
+page-inventory {
```

Comment:
Unless you have an specific plan to use this file, you might want to remove it until you need it (otherwise it could be forgotten forever).